mix.exs

```diff
@@ -0,0 +1,34 @@
+defmodule MarathonEventExporter.Mixfile do
+  use Mix.Project
+
+  def project do
+    [
+      app: :marathon_event_exporter,
+      version: "0.1.0",
+      elixir: "~> 1.5",
+      start_permanent: Mix.env == :prod,
+      deps: deps(),
+      test_coverage: [tool: ExCoveralls],
+      preferred_cli_env: [
+        "coveralls": :test,
+        "coveralls.json": :test,
+        "coveralls.detail": :test,
+      ],
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:httpoison, "~> 0.13"},
```

Comment:
How did you choose an HTTP client implementation? Also, this doesn't actually seem to be used yet?

Reply:
I looked at the five most popular ones and read through their docs. This dep is left over from my toy project which included playing around with HTTP.

I'm currently working on the API client side which uses it, but I'm not sure if that will end up in this branch.